src/app/menus/action-strip/action-strip-paragraph-menu/action-strip-paragraph-menu.component.scss

```
@@ -1,10 +1,12 @@
 ::ng-deep {
```

This should be the other way around
```suggestion
:host ::ng-deep {
```